birdbox/birdbox/settings/base.py

```diff
@@ -395,17 +395,38 @@ def set_whitenoise_headers(headers, path, url):
     "watchman.checks.databases",
 )
 
-# SSL config. Note that SECURE_PROXY_SSL_HEADER has a bearing on SSO redirect URIs
+# Security settings (see `manage.py check --deploy`)
+
 DISABLE_SSL = config("DISABLE_SSL", default=str(DEBUG), parser=bool)  # so default: "False"
-SECURE_SSL_REDIRECT = config("SECURE_SSL_REDIRECT", default=str(not DISABLE_SSL), parser=bool)  # so default: "True"
+SECURE_SSL_REDIRECT = config(
+    "SECURE_SSL_REDIRECT",
+    default=str(not DISABLE_SSL),
+    parser=bool,
+)  # so default: "True"
 if config("USE_SECURE_PROXY_HEADER", default=str(SECURE_SSL_REDIRECT), parser=bool):
     SECURE_PROXY_SSL_HEADER = ("HTTP_X_FORWARDED_PROTO", "https")
-
 SECURE_REDIRECT_EXEMPT = [
     r"^healthz/$",
     r"^readiness/$",
 ]
 
+CSRF_COOKIE_SECURE = True
+CSRF_COOKIE_HTTPONLY = True
+SECURE_BROWSER_XSS_FILTER = True
+SECURE_CONTENT_TYPE_NOSNIFF = True
+SESSION_COOKIE_SECURE = True
```

Comment:
Will this mean sessions don't work for local dev?

Reply:
I thought the same, so I checked beforehand and local dev seems ok - I'm assuming Django checks for localhost/127.0.0.1 and handles it specially